<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/MainWindow/img/microchip.png</normaloff>:/MainWindow/img/microchip.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="WriteOptionsGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Write Options</string>
   </property>
   <widget class="QCheckBox" name="FlashProgramMemorycheckBox">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>30</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>FLASH Program Memory</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ConfigBitsCheckBox">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>60</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Config Bits</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="EepromCheckBox">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>90</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>EEPROM</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
